config/jobs/kubernetes/sig-testing/make-test.yaml

```diff
@@ -41,26 +41,23 @@ presubmits:
     cluster: k8s-infra-prow-build
     skip_branches:
     - release-\d+.\d+ # per-release job
-    labels:
-      preset-service-account: "true"
     optional: true
     always_run: false
     path_alias: k8s.io/kubernetes
     spec:
+      # let's see if we can run unit tests as a non-root user
+      securityContext:
```

Comment:
it appears we can't write to the code mount when we just set the test container, so that's an issue.
setting this on the pod level should default it for all containers.